README.md

```diff
@@ -181,7 +181,7 @@ See docker-compose.yml for components.
 ##### Redirections
 - `MARKETPLACE_BASE_URL`: `Url = "https://marketplace.eosc-portal.eu/"` - marketplace base url (used to generate links back to MP).
 - `EOSC_COMMONS_URL`: `Url = "https://s3.cloud.cyfronet.pl/eosc-portal-common/"` - Base URL to eosc commons.
-- `EOSC_COMMONS_ENV`: `str = "production"` - Environment used to load eosc commons. Together with `EOSC_COMMONS_URL` two assets are loaded:
+- `EOSC_COMMONS_ENV`: `str = "beta"` - Environment used to load eosc commons. Together with `EOSC_COMMONS_URL` two assets are loaded:
```

Comment:
you can add information here that this is an enum, and can be one of two values:
 'production' | 'beta'